packages/example/react/src/App.tsx

```diff
@@ -1,26 +1,17 @@
 import { GlowWalletAdapter } from '@solana/wallet-adapter-glow';
 import { PhantomWalletAdapter } from '@solana/wallet-adapter-phantom';
 import { registerWalletAdapter, SOLANA_MAINNET_CHAIN } from '@solana/wallet-standard';
-import { useWallets, WalletProvider } from '@wallet-standard/react';
-import type { FC, ReactNode } from 'react';
+import { useWallets } from '@wallet-standard/react';
+import type { FC } from 'react';
 import React, { useEffect } from 'react';
 
 export const App: FC = () => {
-    return (
-        <Context>
-            <Content />
-        </Context>
-    );
-};
-
-const Context: FC<{ children: NonNullable<ReactNode> }> = ({ children }) => {
     useEffect(() => {
         const adapters = [new PhantomWalletAdapter(), new GlowWalletAdapter()];
         const destructors = adapters.map((adapter) => registerWalletAdapter(adapter, SOLANA_MAINNET_CHAIN));
         return () => destructors.forEach((destroy) => destroy());
     }, []);
-
-    return <WalletProvider>{children}</WalletProvider>;
+    return <Content />;
```

Comment:
Does this mean you don't need a provider wrapper any more? 🔥

Reply:
![image.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/lE1zD7Jpg6mWNv3djNKZ/467f2624-899d-4b1a-87d1-aabd10e11a32.png)